index.html

```diff
@@ -7,275 +7,221 @@
     <meta http-equiv="X-UA-Compatible" menu="ie=edge">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <script src='https://kit.fontawesome.com/a076d05399.js'></script>
-    <link rel="stylesheet" href="grid.min.css">
-    <link href="https://fonts.googleapis.com/css?family=Poppins:300,400&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Poppins:100,200,300,400&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
     <title>iPaidThat, Project X</title>
 </head>
 
 <body>
-    <div class="container">
-        <nav class="c-nav">
-            <div class="c-nav__inner">
-                <div class="c-nav__section">
-                    <ul class="ul">
-                        <li>
-                            <select class="c-nav__dropdown">
-                                <option value="Drop 1">Drop 1</option>
-                                <option value="Drop 2">Drop 2</option>
-                                <option value="Drop 3">Drop 3</option>
-                            </select>
-                        </li>
-                        <li>Item 2</li>
-                        <li>Item 3</li>
-                    </ul>
-                </div>
-                <div class="c-nav__section">
-                    <h1><em>iPaidThat</em></h1>
-                </div>
-                <div class="c-nav__section">
-                    <ul class="ul">
-                        <li>Item 1</li>
-                        <li><button class="button button--round">Item 2</button></li>
-                    </ul>
-                </div>
+
+    <nav class="c-nav">
+        <div class="c-nav__inner">
+            <div class="c-nav__section">
+                <ul class="ul">
+                    <li>
+                        <select class="c-nav__dropdown">
+                            <option value="Drop 1">Drop 1</option>
+                            <option value="Drop 2">Drop 2</option>
+                            <option value="Drop 3">Drop 3</option>
+                        </select>
+                    </li>
+                    <li>Item 2</li>
+                    <li>Item 3</li>
+                </ul>
+            </div>
+            <div class="c-nav__section">
+                <h1><em>iPaidThat</em></h1>
+            </div>
+            <div class="c-nav__section">
+                <ul class="ul">
+                    <li>Item 1</li>
+                    <li><button class="button button--round">Item 2</button></li>
```

Comment:
Is this `button--round` the same as...

Reply:
No this is the white button in the nav bar, I'll rename it for clarity